Automattic/wp-calypso
# Plans redesign: Available and missing features

#### Changes proposed in this Pull Request

This PR implements the experiment that redesigns the signup flow plans step. Users in the treatment group will see the new feature comparison section for each plan in the signup plans step.

NOTE: 
1. This PR uses the changes in https://github.com/Automattic/wp-calypso/pull/50005 as the base branch and is built on top of it.

**Important links**
1. [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1362%3A813)
2. pau2Xa-2yE-p2 for the project thread.


**Annual plans view with free subdomain selected (the first feature item is a generic free domain message)**

![calypso localhost_3000_start_plans (5)](https://user-images.githubusercontent.com/1269602/108342053-5cf41a80-7200-11eb-808a-974d928bc6db.png)

**Annual plans view with a custom domain selected (the first feature item mentions the selected domain)**

![calypso localhost_3000_start_plans (7)](https://user-images.githubusercontent.com/1269602/108342101-6d0bfa00-7200-11eb-9d03-656a5f0fbc23.png)

**Monthly plans view with a free subdomain selected (Live chat and free domain mentioned as unavailable)**

![calypso localhost_3000_start_plans (6)](https://user-images.githubusercontent.com/1269602/108342132-74cb9e80-7200-11eb-88ce-707857c54e43.png)


**Monthly plans view with a custom domain selected (Live chat and free domain mentioned as unavailable)**

![calypso localhost_3000_start_plans (8)](https://user-images.githubusercontent.com/1269602/108342156-7bf2ac80-7200-11eb-9264-72de3866f61a.png)


#### Testing instructions



* Open http://calypso.localhost:3000/ and assign yourself to treatment of the `signup_plans_step_redesign_v1` experiment in the Abacus tool.
* Begin signup at `/start` and verify that the plans step UI matches the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1362%3A813). Verify for both free subdomain and custom domain selections.
* Verify that the UI matches the Figma design for both annual and monthly plan views.
* Confirm that in Calypso's /plans view, the UI is unchanged.
* Also confirm that the launch flow UI is unchanged.
* Verify that control experience is unchanged.



Related to https://github.com/Automattic/wp-calypso/pull/49877
